go/tools/builders/importcfg.go

```diff
@@ -71,7 +71,11 @@ func checkImports(files []fileInfo, archives []archive, stdPackageListPath strin
 			importAliasToArchive[imp] = arc
 		}
 	}
-
+	// Construct recompileInternalDeps as a map to check if there are imports that are disallowed.
+	recompileInternalDepMap := make(map[string]bool)
```

Comment:
```suggestion
	recompileInternalDepMap := make(map[string]struct{})
```
Making the value `struct{}` is less ambiguous. It only has one value, so people don't need to guess what `true` or `false` in the map means.